### PR TITLE
Updated docs for csm operator

### PR DESCRIPTION
**Pull request by tdawe**
# Description
Update csm operator docs

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Comment by mjsdell** [comment]
Adding Ray for his review

**Review by rsedlock1958 — COMMENTED** [review]
I reviewed this. The changes look fine.

File: content/docs/deployment/csmoperator/_index.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Dell CSM Operator can be installed manually or via Operator Hub.
 #### Operator Installation on a cluster without OLM
 
 1. Clone the [Dell CSM Operator repository](https://github.com/dell/csm-operator).
+2. (Optional) If using a local Docker image, edit the `deploy/operator.yaml` file and set the image name for the CSM Operator Deployment.
```
Comment (nb950) [inline content/docs/deployment/csmoperator/_index.md:41]: (Optional) If using a local Docker image,
can be removed
edit the `deploy/operator.yaml` file and set the image name for the CSM Operator Deployment as required

File: content/docs/deployment/csmoperator/troubleshooting/_index.md [hunk 2]
```diff
@@ -23,4 +22,7 @@ The Dell CSM Operator is not fully compliant with the OperatorHub React UI eleme
 Detailed logs of the CSM Operator can be displayed using the following command:
 ```
 kubectl logs <csm-operator-controller-podname> -n <namespace>
-```
\ No newline at end of file
+```
+
+### My Dell CSI Driver install failed. How do I fix it?
```
Comment (nb950) [inline content/docs/deployment/csmoperator/troubleshooting/_index.md:27]: kubectl describe csm test-isilon -n test-isilon
refer  kubectl logs dell-csm-operator-controller-manager-559dc5b6bf-pjlsv -n dell-csm-operator for operator logs and errors during driver install.

refer the status and events section . If status has failed pods , refer the driver documentation to troubleshoot driver failures.
Status:
  Controller Status:
    Available:  1
    Desired:    2
    Failed:     1
  Node Status:
    Available:  1
    Desired:    2
    Failed:     1
  State:        Failed

Events 
 Warning  Updated    67s (x15 over 2m4s)  csm   (combined from similar events): at 1646848059520359167 Pod error details ControllerError: ErrImagePull= pull access denied for dellem/csi-isilon, repository does not exist or may require 'docker login': denied: requested access to the resource is denied, Daemonseterror: ErrImagePull= pull access denied for dellem/csi-isilon, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
the above event shows  dellem/csi-isilon does not exist , to fix kubectl edit the csm and update to right image

Comment (tdawe) [reply content/docs/deployment/csmoperator/troubleshooting/_index.md:27]: Added this

File: content/docs/deployment/csmoperator/_index.md [hunk 3]
```diff
@@ -170,13 +171,10 @@ Once the driver `Custom Resource (CR)` is created, you can verify the installati
 
 *  Check if ContainerStorageModule CR is created successfully using the command below:
     ```
-    $ kubectl get csm -n <driver-namespace>
+    $ kubectl get csm/<name-of-custom-resource> -n <driver-namespace> -o yaml
     ```
-* Check the status of the CR to verify if the driver installation is successful.
+* Check the status of the CR to verify if the driver installation is in the `Succeeded` state. If the status is not `Succeeded`, see the [Troubleshooting guide](./troubleshooting/) for more information.
```
Comment (shanmydell) [inline content/docs/deployment/csmoperator/_index.md:176]: @tdawe : This link will not work with Hugo

**Review by shanmydell — CHANGES_REQUESTED** [review]

File: content/docs/deployment/csmoperator/_index.md (hunk 3 above)
Comment (tdawe) [reply content/docs/deployment/csmoperator/_index.md:176]: @shanmydell it worked on my Hugo instance version `hugo v0.89.0-DEV+extended`. Is there another version being used for our live site?

**Review by bharathsreekanth — DISMISSED** [review]

File: content/docs/deployment/csmoperator/_index.md [hunk 4]
```diff
@@ -38,6 +38,7 @@ Dell CSM Operator can be installed manually or via Operator Hub.
 #### Operator Installation on a cluster without OLM
 
 1. Clone the [Dell CSM Operator repository](https://github.com/dell/csm-operator).
+2. (Optional) If using a local Docker image, edit the `deploy/operator.yaml` file and set the image name for the CSM Operator Deployment.
 2. `cd csm-operator`
```
Comment (shanmydell) [inline content/docs/deployment/csmoperator/_index.md:42]: unless we cd to the operator , we cannot run the command below in point 3 @tdawe , can we add it back

File: content/docs/deployment/csmoperator/_index.md (hunk 3 above)
Comment (shanmydell) [reply content/docs/deployment/csmoperator/_index.md:176]: @tdawe : It ll goto the page , but not the section "troubleshooting/#my-dell-csi-driver-install-failed-how-do-i-fix-it"

File: content/docs/deployment/csmoperator/_index.md (hunk 4 above)
Comment (tdawe) [reply content/docs/deployment/csmoperator/_index.md:42]: Ok re-ordering these steps

File: content/docs/deployment/csmoperator/_index.md (hunk 3 above)
Comment (tdawe) [reply content/docs/deployment/csmoperator/_index.md:176]: Ok right, updated to go to proper section

**Review by shanmydell — APPROVED** [review]

**Review by bharathsreekanth — APPROVED** [review]
Thanks @tdawe for all the help.

**Comment by bharathsreekanth** [comment]
thanks @shanmydell for all the input.

**Pull request merged**